### DISCUSSION TITLE
Neotest hangs entire neovim instance after latest update

### DISCUSSION THREAD

**Discussion by mattd-tg** [LazyVim/LazyVim#5154] [Q&A]
Trying to figure out how best to debug this and wondering if anybody else is experiencing the same? I have `neotest` configured and often run all tests in the open buffer. I have the `neotest-jest` adapter included and that seems to cause everything to hang. I have not yet tested other languages to see if it is specific to that adapter. If I remove the adapter completely and just have nothing, it fixes the freezing but obviously no tests are recognized.

Any `neotest` commands completely hang `neovim` and I have to force quit the instance to recover.

Nothing gets written to `~/.local/state/nvim/neotest.log` before hanging

Tried going back to `nvim-cmp` from `blink` since that was one of the major changes, but that did not fix the issue. Possibly one of the other `snack` migrations causing it?

```
NVIM v0.10.2
Build type: Release
LuaJIT 2.1.1727870382
```

**Comment by crisidev** [comment]
I am having the same issue with rust tests. And the same happens with nightly neovim.

**Reply by crisidev** [reply]
It seems to be related to this plugin issue: https://github.com/willothy/flatten.nvim/issues/106

**Comment by dpetka2001** [comment]
There is an open issue at neotest repo, so i'd suggest starting there. Someone even tried a hotfix in his forked repo to disable `subprocess`, but that didn't work for everyone.
